odl/solvers/functional/functional.py

```diff
@@ -225,6 +225,48 @@ def translated(self, shift):
         """
         return FunctionalTranslation(self, shift)
 
+    def bregman(self, point, subgrad=None):
+        """Return the Bregman distance functional.
+
+        Parameters
+        ----------
+        point : element of ``functional.domain``
+            Point from which to define the Bregman distance.
+        subgrad : `Operator`, optional
```

Comment:
We can go for gradient, no problem with me :-)

Reply:
Actually, trying to go through and making this change it turns out that we would probably like to distinguish it from the `gradient`. Because this is the (sub)gradient of the "underlying" functional. Meanwhile, the `BregmanDistance` is going to have a `gradient` of its own (which is a translated version of what is now called `subgrad`, see [line 1505](https://github.com/odlgroup/odl/pull/1267/files/3f2a8c2dfb158a059a214858b3d5ecce8bb640a9#diff-4086f82398c755f2cf1063ced9ec1c93R1505)), and we don't want to confusing these two.

Any other naming suggestions? `gradient_op`? Or stick with `subgrad`?

Reply:
I'm not in favor of changing `subgrad` to `gradient` anyway, it's less precise IMO. To make clear that it's a property of the underlying functional, we could call it `func_subgrad` or so.